pkg/sql/opt/testutils/opttester/opt_tester.go

```diff
@@ -2326,3 +2336,54 @@ func (ot *OptTester) ExecBuild(f exec.Factory, mem *memo.Memo, expr opt.Expr) (e
 	)
 	return bld.Build()
 }
+
+func (ot *OptTester) StatementBundle(tb testing.TB) error {
+	if ot.Flags.File == "" {
+		return errors.New("statement-bundle requires file argument")
+	}
+	dir := ot.Flags.File
+	f, err := os.Open(dir)
+	if err != nil {
+		// try relative path
+		dir = datapathutils.TestDataPath(tb, dir)
+		var err2 error
+		f, err2 = os.Open(dir)
+		if err2 != nil {
+			return errors.CombineErrors(err, err2)
+		}
+	}
+	s, err := f.Stat()
+	if err != nil {
+		return err
+	}
+	// TODO(cucaroach): support direct from zip
+	if !s.IsDir() {
+		return errors.New("statement-bundle argument must be a directory")
+	}
+	schema, err := os.ReadFile(filepath.Join(dir, "schema.sql"))
+	if err != nil {
+		return err
+	}
+	testCatalog, ok := ot.catalog.(*testcat.Catalog)
+	if !ok {
+		return errors.New("exec-ddl can only be used with TestCatalog")
```

Comment:
nit: I don't think this error makes sense because there is no `exec-ddl` command in this case.